pkg/pool-weighted/contracts/smart/IndexPool.sol

```diff
@@ -111,7 +117,9 @@ contract IndexPool is BaseWeightedPool, ReentrancyGuard {
         returns (
             uint256 startTime,
             uint256 endTime,
-            uint256[] memory endWeights
+            uint256[] memory endWeights,
+            uint256[] memory startWeights,
```

Comment:
Why do we need all this info?

Reply:
Specifically startWeights

Reply:
It's not necessary for any contract logic (as is the whole function `getGradualWeightUpdateParams`). Essentially the whole function just exposes the tokenStates. Adding these return values enables to test contract logic, while doing no harm and potentially even providing additional valuable information to the outside world.

Reply:
Well, from one side, from other side it's eating gas, I don't think that it eats a lot, but still enough, it's not critical though, let's remove it in the end if we're not going to use it....